test-result-summary-client/package.json

```diff
@@ -41,7 +41,7 @@
   "devDependencies": {
     "cypress": "^8.6.0",
     "react-app-rewired": "^2.1.8",
-    "react-scripts": "^3.4.4"
+    "react-scripts": "^3.0.1"
```

Comment:
We should upgrade to a newer version, not an older version. ;) I am also expecting changes in package-lock.json (auto-generated). On the second thought, let's create a separate PR for module upgrades. Thanks.

Reply:
Sounds good. I'll remove this commit from this PR